src/cryptonote_core/blockchain.cpp

```diff
@@ -128,7 +128,8 @@ static const struct
 	{2, 5150, 0, 1497181713},
 	{3, 103580, 0, 1522540800}, // April 01, 2018
 	{4, 123575, 0, 1529873000},
-	{5, 129750, 0, 1532782050}
+	{5, 129750, 0, 1532782050},
+	{6, 130425, 0, 1532782050}
```

Comment:
Timestamp must be greater than the fork time before: https://github.com/ryo-currency/ryo-currency/blob/df6ec0a7e6fc65e7c92b859ddcf57615bd87170a/src/cryptonote_basic/hardfork.cpp#L100